site/source/docs/javascript_usage.md

```diff
@@ -26,15 +26,9 @@ The following code imports EmbarkJS:
 import EmbarkJS from './embarkArtifacts/embarkjs';
 ```
 
-## Connecting to a Blockchain node
-
-EmbarkJS offers different APIs for various decentralized services. Some of them can be used right away, others require Embark to do some initial work first. For example, in order to use any [methods of Smart Contract instances](/docs/contracts_javascript.html), it's important that EmbarkJS is actually connected to a blockchain node.
-
-This can be done using the `EmbarkJS.Blockchain.connect()` API. This method takes a configuration object which specifies what node to connect to, and receives a callback that will be called once Embark has connected successfully.
-
 ### Installing a blockchain connector
 
-Before the `EmbarkJS.Blockchain.connect()` API can be used, we need to install a blockchain connector plugin. This is required because Embark doesn't have a strong opinion about which connector library we should use. Some people like to use `web3.js`, others prefer libraries like `ethers.js`. Once we know which library to use, we'll have to install a dedicated connector plugin. The Embark project maintains a blockchain connector for the `web3.js` library called `embarkjs-connector-web3`.
+Before the Embark can connect to the blockchain, we need to install a blockchain connector plugin. This is required because Embark doesn't have a strong opinion about which connector library we should use. Some people like to use `web3.js`, others prefer libraries like `ethers.js`. Once we know which library to use, we'll have to install a dedicated connector plugin. The Embark project maintains a blockchain connector for the `web3.js` library called `embarkjs-connector-web3`.
```

Comment:
Just a reminder that if we get #1661 merged then we'll need to remove the instructions re: `embarkjs-connector-web3` or qualify them with respect to embark older than `v4.1.0`.

Also note that @iurimatias is presently working on a larger refactor branched off #1661, which may takes its place.

Reply:
Good point @michaelsbradleyjr that actually means #1661 has to introduce those docs changes as well.